R/estimateEvenness.R

```diff
@@ -0,0 +1,177 @@
+#' Estimate Evenness
+#'
+#' Several functions for calculation of evenness estimates are available.
+#'
+#' These includes the \sQuote{Camargo}, \sQuote{Pielou}, \sQuote{Simpson},
+#' \sQuote{Evar} and \sQuote{Bulla} evenness measures.
+#'
+#' @param x a \code{\link{SummarizedExperiment}} object
+#'
+#' @param abund_values the name of the assay used for calculation of the
+#'   sample-wise estimates
+#'
+#' @param index a evenness measurement
+#'
+#' @param name a name for the column of the colData the results should be stored
+#'   in.
+#'
+#' @param threshold a numeric threshold. assay values below or equal to this
```

Comment:
I find this confusing that lowercase is used when starting sentences. Or is some convention to use in manpages?

Reply:
changed